src/test/scala/com/amazon/deequ/analyzers/StateProviderTest.scala

```diff
@@ -101,50 +101,6 @@ class StateProviderTest extends WordSpec with Matchers with SparkContextSpec wit
 
       assertCorrectlyApproxQuantileState(provider, provider, ApproxQuantile("price", 0.5), data)
     }
```

Comment:
why is this deleted?

Reply:
I haven't make changes to StateProvider.scala or  StateProviderTest.scala. This is probably because I didn't rebase the latest master. Will check again after rebase.